code/modules/research/designs/mecha_equipment.dm

```diff
@@ -117,18 +123,33 @@
 	name = "MkIV ion heavy cannon"
 	build_path = /obj/item/mecha_parts/mecha_equipment/ranged_weapon/energy/ion
 
-/datum/design/research/item/mecha/weapon/sonic
```

Comment:
Why are removing this

Reply:
Didn't mean to. Simply something I missed out in my code. Will resolve that now.